src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/AnonymizerConfigurations/ProcessingErrorsOption.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Fhir.Anonymizer.Shared.Core.AnonymizerConfigurations
+{
+    public enum ProcessingErrorsOption
+    {
+        Raise, // Invalid processing will raise an exception.
+        Skip,  // Invalid processing will return null.
+        // Ignore Invalid processing will return input.
```

Comment:
nit: consider moving the comment for `ignore` to `AnonymizerConfiguration`. The property is set to `ignore` there instead of in this enum.

Reply:
Ignore might be the future option in this enum.